perllib/FixMyStreet/App/Form/Waste/Garden/Renew.pm

```diff
@@ -29,12 +29,17 @@ has_page intro => (
 
         my $max_bins = $data->{max_bins};
         my %bin_params = ( default => $data->{bins}, range_end => $max_bins );
+
+        my ($name, $email, $phone) = ('', '', '');
+        if ($c->user_exists) {
+            ($name, $email, $phone) = ($c->user->name, $c->user->email, $c->user->phone);
+        }
```

Comment:
This defaults to the staff account details. Is that what we want?

Reply:
Ah, probably not, should do whatever the subscribe form does the same I guess.

Reply:
So rare, should just make them type it in I think probably, like subscribe.

Reply:
Yes, that makes sense.